### DISCUSSION TITLE
Back up downloads folder

### DISCUSSION THREAD

**Discussion by corserp** [apache/incubator-mxnet#19165] [Q&A]
What does incubator do?

How to use it?

**Comment by szha** [comment]
Hi, the question is not making sense to me and it doesn't seem related to the title either. please explain

**Reply by corserp** [reply]
I was wondering why anyone would use incubator or mxnet when there are other ansibles available to the public...

I use atom.

**Comment by gundalow** [comment]
@szha This account seems to be spamming various Ansible repos as well (which I'm the maintainer of) I've reported them to GitHub, suggest you close/delete

**Reply by corserp** [reply]
I think you have the wrong impression @gundalow 
please provide spam if you're really the "owner".
